mahta/CSS/CSS-box-model/styles4.css

```diff
@@ -0,0 +1,38 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(5, 20px);
+  grid-gap: 8px;
+}
+.container div {
+  background-color: red;
+  height: 20px;
+  width: 20px;
+}
+
+.containerY {
+  display: flex;
+  align-items: center;
+}
+.containerY div {
+background-color: yellow;
+display: flex;
+align-items: center;
+margin-right: 48px;
+height: 16px;
```

Comment:
These small widths cause the text to seem outside of the container, while it's not.

Reply:
I'm not certain what I must do about this comment, I tried deleting or changing each of the lines above. It only caused the output to ruin!
However, by deleting containerY class for p and div(lines 24 to 26), the text would start from inside the square, if that's what we are looking for.

Reply:
Or perhaps, am I missing something?

Reply:
You just need to increase the width and height of the squares, so the text fits the square.

Reply:
the numbers may not be accurate but i think it's solved.